test/relaying.js

```diff
@@ -420,6 +444,17 @@ test('relay connections through node, client and server side', async function (t
     })
     .end('hello world')
 
+  bSocket.relay
+    .on('relay', () => {
+      lc.pass('client socket is being relayed')
+      lc.is(bSocket.relay.relaying, true)
+      lc.alike(bSocket.relay.remotePublicKey, bServer.publicKey)
+      lc.alike(bSocket.relay.relayThrough, aServer.publicKey)
+    })
+    .on('abort', () => {
+      lc.fail('client relay aborted')
+    })
```

Comment:
AFAICT this is the only `relaying.js` test that actually relays. All other tests abort the relay.